pkg/sources/gitlab/gitlab.go

```diff
@@ -81,6 +82,56 @@ func (s *Source) JobID() sources.JobID {
 	return s.jobID
 }
 
+// globRepoFilter is a wrapper around cache.Cache that filters out repos
+// based on include and exclude globs.
+type globRepoFilter struct {
+	include, exclude []glob.Glob
+}
+
+func newGlobRepoFilter(ctx context.Context, include, exclude []string, onCompileErr func(err error, pattern string)) *globRepoFilter {
```

Comment:
it doesn't look like `ctx` is used here

Reply:
Thanks for highlighting. Right, Its not being used. Let me clean it.

Reply:
Removed.